frontend/src/tbpro/elements/SwitchToggle.vue

```diff
@@ -29,6 +29,15 @@ const toggleState = () => {
     emit('changed', state.value);
   }
 };
+
+// Update state if parent changes active prop
+// e.g. after initializing async data
+watch(
+  () => props.active,
+  () => {
+    state.value = props.active;
+  }
+);
```

Comment:
I think you might be able to {active} = toRef(props) instead of this.

Reply:
Actually I tried `const state = toRef(props.active);` but it won't properly initialize state on page load... 😅 Or I'm just to stupid to properly implement this 😂 Maybe I oversaw sth?

Reply:
Oh I think you need to toRef props itself and destructing assign active.

so `{ active } = toRef(props);`

but i can take a look on tuesday!

Reply:
Ah, you mean `toRefs()` (with an 's')? OR actually `toRef()`? I tried the former too but it complained about being read-only when updating that ref 😂

Reply:
This works for now 😄

Reply:
haha alright! thanks